src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,114 @@
+package duke;
+
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import exception.DukeErrorException;
+import exception.InvalidDeadlineException;
+import exception.InvalidEventException;
+import exception.InvalidTodoException;
+import ui.Ui;
+
+
+/**
+ * Main class for the Duke application
+ */
+public class Duke {
+    // Scanner to read input
+    private Scanner readSc;
+    // Scanner to give input
+    private Scanner inputSc;
+    // List of tasks
+    private ArrayList<Task> tasks;
+    // Ui object to print out displays
+    private Ui ui;
+    // Storage object to handle saving to file
+    private Storage storage;
+    // TaskList object to handle insertion, deletion, etc of tasks
+    private TaskList tl;
+    // Parser object to process inputs and commands
+    private Parser parser;
```

Comment:
You may want to change these comments to single line Javadoc comments e.g. `/** Scanner to read input */`.
This would also apply to any class members in other classes 😅 . (You can look for the example "Javadoc of class members can be specified on a single line as follows" in the [conventions page](https://se-education.org/guides/conventions/java/intermediate.html#comments)! 😄 )

Reply:
Alright, thanks for the input! Will surely take a look.